hs_model_program/static/js/model_program.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Created by tonycastronova on 10/12/15.
+ */
+
+$alert_error = '<div class="alert alert-danger" id="error-alert"> \
+                    <button type="button" class="close" data-dismiss="alert">x</button> \
+                    <strong>Error </strong> \
+                    failed to retrieve metadata for this resource.\
+                </div>'
+
+
+$(document).ready(function () {
+
+    // check to see if the view is in edit mode
+    if(document.getElementById("id-mpmetadata") != null)
+        // load jquery generated elements
+        loadWidgets();
+
+        // initialize datepicker
+        $( "#modelReleaseDate_picker" ).datepicker({
+            // bind a table view to the on close event
+            onSelect: function (event) {
+                onChange_releaseDate(event);
+            }
+        });
+});
+
+/**
+ * Sets the hidden date field base on the datepicker.  The hidden field is what is submitted.
+ */
+function onChange_releaseDate(event){
+    var date = document.getElementById('id_modelReleaseDate');
+    var date_picker = document.getElementById("modelReleaseDate_picker");
+    date.value = date_picker.value;
+
+    // activate the save button
+    $("#resourceSpecificTab").find('.btn-primary').show();
+}
+
+/**
+ * Builds jQuery multiselect listboxes
+ */
+function loadWidgets(){
+
+    // get the resource id from its url
+    var url = document.URL;
+    var url_array = url.split('/');
+    var shortid = url_array[url_array.length - 2];
+
+    // call django view to get model program metadata files
+    $.ajax({
+        type: "GET",
+        url: '/hsapi/_internal/get-model-metadata/',
+        data: {resource_id: shortid},
+        success: function (data) {
+
+            // get the fieldset items
+            var fs = document.getElementById("id-mpmetadata").getElementsByTagName('fieldset')[0];
```

Comment:
This has been replaced with a jQuery selector 8a1e045f5e011aedd64c7572422b62de34b7e402